test/testhelpers/nfs_storageclass.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: nfs-server
+provisioner: example.com/nfs
+parameters:
+  archiveOnDelete: "false"
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
```

Comment:
It looks like Github UI is complaining about this file missing a trailing newline.

Reply:
Yeah, that's a new thing. Also there's lines modded in here in manual_install_tests that I did not change.